tests/functional/sync_v3_sign_up.js

```diff
@@ -11,16 +11,18 @@ const selectors = require('./lib/selectors');
 const uaStrings = require('./lib/ua-strings');
 
 const config = intern._config;
-const SIGNUP_PAGE_URL = `${config.fxaContentRoot}signup?context=fx_desktop_v3&service=sync&forceAboutAccounts=true&automatedBrowser=true`;
+const SIGNUP_PAGE_URL = `${config.fxaContentRoot}signup?context=fx_desktop_v3&service=sync`;
+const ENTER_EMAIL_PAGE_URL = `${config.fxaContentRoot}?context=fx_desktop_v3&service=sync&automatedBrowser=true`;
 
 let email;
 const PASSWORD = '12345678';
```

Comment:
This reminds me that when we graduate the password experiment, we will need to update all the functional tests to use stronger passwords 😳

Reply:
Oh dear lord, I hadn't thought of that.